module-services/service-cellular/src/ServiceCellularPriv.cpp

```diff
@@ -550,6 +552,52 @@ namespace cellular::internal
         owner->csqTimer.start();
     }
 
+    void ServiceCellularPriv::initUSSDHandler()
+    {
+        ussdHandler->onOpenPushSession = [this]() -> bool {
+            auto channel = owner->cmux->get(CellularMux::Channel::Commands);
+            if (channel == nullptr) {
+                LOG_ERROR("No cmux channel provided!");
+                return false;
+            }
+            return channel->cmd(at::AT::CUSD_OPEN_SESSION).code == at::Result::Code::OK;
+        };
+
+        ussdHandler->onAbortSession = [this]() -> bool {
+            auto channel = owner->cmux->get(CellularMux::Channel::Commands);
+            if (channel == nullptr) {
+                LOG_ERROR("No cmux channel provided!");
+                return false;
+            }
+            return channel->cmd(at::AT::CUSD_CLOSE_SESSION).code == at::Result::Code::OK;
+        };
+
+        ussdHandler->onSendUssdCode = [this](const std::string &request) -> bool {
+            constexpr uint32_t commandTimeout(120000);
+
+            auto channel = owner->cmux->get(CellularMux::Channel::Commands);
+            if (channel == nullptr) {
+                LOG_ERROR("No cmux channel provided!");
+                return false;
+            }
+
+            channel->cmd(at::AT::SMS_GSM);
+            std::string command = at::factory(at::AT::CUSD_SEND) + request + ",15";
```

Comment:
can't this `,15` be moved to a variable?

Reply:
done